index.d.ts

```diff
@@ -0,0 +1,50 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string. If the variable is
+   * not set and a default was specified, the default value is returned,
+   * otherwise an error is thrown.
+   */
```

Comment:
We should not use this syntax, there is good reason not to. In most editors, editors can be configured to pick up function comments if they have `jsdoc` / `tstoc` syntax.

Here is what it looks like in `sublime`, it works in `vim` and `vscode` as well.
![image](https://user-images.githubusercontent.com/6983492/28175685-c462ea94-67a9-11e7-9dca-5f65749d2f14.png)